plugin.py

```diff
@@ -820,6 +877,14 @@ def updateDeviceSettings(self, devicename, devicetype, config):
                 oldconfigdict = json.loads(device.Options['config'])
             except (ValueError, KeyError, TypeError) as e:
                 pass
+            # Sensor support
+            # Correction for subsequent messages
```

Comment:
Please elaborate this comment to explain what's going on

Reply:
BLE sensors send the discovery info in several MQTT messages per sensors (Temperature, Humidity, RSSI, Battery) component, and this requires state tracking between different MQTT messages